spec/lib/chatops/controller_spec.rb

```diff
@@ -60,7 +60,7 @@ def ensure_app_given
   end
 
   def rails_flexible_post(path, outer_params, jsonrpc_params = nil)
-    if Rails.version.starts_with?("4")
+    if Rails.version.start_with?("4")
```

Comment:
`if defined?(Rails)` necessary here?

Reply:
From checking the ActionPack source, it appears `Rails` will be defined even if all of Rails isn't loaded. https://github.com/rails/rails/blob/main/actionpack/lib/action_dispatch/railtie.rb#L7